### PR TITLE
Added tensorboard

### PR DESCRIPTION
**Pull request by MihailSalnikov**
Hello, 
I added tensorboard for visualization, if it's ok, please add it to project.

### CODE REVIEW

**Comment by DropEdge** [comment]
Hi MihailSalnikov,
     Thanks a lot for your code of better visualization! We'll review the code and re-run all experiments to ensure reproducibility.  If everything is OK, we'll merge it into master asap.

File: requirements.txt [hunk 1]
```diff
@@ -22,4 +22,4 @@ scipy==1.2.1
 six==1.10.0
 torch==0.4.0
 torchvision==0.2.1
-
+tensorboard==2.1.0
```
Comment (xuty-007) [inline requirements.txt]: It is incompatible with torch 0.4.0. It is better to install both tensorboard and tensorboardX==2.0
Full requirements.txt:
```
asn1crypto==0.22.0
certifi==2017.4.17
cffi==1.10.0
decorator==4.4.0
idna==2.6
networkx==2.3
numpy==1.14.3
olefile==0.45.1
packaging==16.8
pandas==0.24.2
Pillow==4.2.1
pycosat==0.6.3
pycparser==2.18
pycrypto==2.6.1
pyOpenSSL==17.0.0
pyparsing==2.2.0
python-dateutil==2.8.0
pytz==2019.1
requests==2.21.0
scikit-learn==0.20.3
scipy==1.2.1
six==1.10.0
torch==0.4.0
torchvision==0.2.1
tensorboard==2.1.0
tensorboardX==2.0
```

File: src/train_new.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 import torch
 import torch.nn.functional as F
 import torch.optim as optim
+from torch.utils.tensorboard import SummaryWriter
```
Comment (xuty-007) [inline src/train_new.py]: torch 0.4.0 does not include tensorboard. Please use tensorboardX instead:
```
from tensorboardX import SummaryWriter
```

File: requirements.txt (hunk 1 above)
Comment (MihailSalnikov) [reply requirements.txt]: Done

File: src/train_new.py (hunk 2 above)
Comment (MihailSalnikov) [reply src/train_new.py]: Done

**Comment by xuty-007** [comment]
Thank you for your work. I've checked all the code and revisions. Everything is right. We'll merge it soon.

**Pull request merged**